explainer.md

```diff
@@ -79,21 +76,23 @@ Now rather than the browser having to block scrolling whenever there is any touc
 
-## Removing the need to cancel touch events
+## Removing the need to cancel events
+
+There are scenarios where an author may intentionally want to disable scrolling by cancelling touch or wheel events. These include:
+ * Panning a map
+ * Full-page/full-screen games
+ In these cases, the current browser behavior (which prevents scrolling optimization) is perfectly adequate, since scrolling itself is being prevented.
 
-In general touch listeners should always be `passive` unless you know they need to block scrolling.  In a number of common scenarios the `passive` option can be added (with appropriate feature detection) without any other changes, eg:
+ However, in a number of common scenarios events don't need to block scrolling, eg:
  * User activity monitoring which just wants to know when the user was last active
  * `touchstart` handlers that hide some active UI (like tooltips)
  * `touchstart` and `touchend` handlers that style UI elements (without suppressing the `click` event).
+ For these scenarios, the `passive` option can be added (with appropriate feature detection) without any other code changes, resulting in a significantly smoother scrolling experience.
 
-And of course there are scenarios where there is no need to use a `passive` listener because the listener intentionally disables scrolling in all cases, eg:
- * Panning a map
- * Full-page games
-
-But there are a few more complicated scenarios where the handler really wants to suppress scrolling some cases but not in others.  eg:
+There are a few more complicated scenarios where the handler only wants to suppress scrolling under certain conditions, eg:
  * Swiping horizontally to rotate a carousel, dismiss an item or reveal a drawer, while still permitting vertical scrolling.
    * In this case, use [touch-action](https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action) to declaratively disable scrolling along one axis without having to call `preventDefault()`.
-   * To continue to work correctly in all browsers, calls to `preventDefault` should be conditional on the lack of support for the particular `touch-action` rule being used (note that Safari 9 supports `touch-action: manipulation` but not the other values).
+   * To continue to work correctly in all browsers, calls to `preventDefault` should be conditional on the lack of support for the particular `touch-action` rule being used (note that Safari 9 currently only supports `touch-action: manipulation`).
  * Event delegation patterns where the code that adds the listener won't know if the consumer will cancel the event.
    * One option here is to do delegation separately for passive and non-passive listeners (as if they were different event types entirely).
    * It's also possible to leverage `touch-action` as above (treating Touch Events as you would [Pointer Events](https://w3c.github.io/pointerevents/).
```

Comment:
Since we're now including wheel here, it may be worth adding another bullet for the limitation there.  Eg. "A UI element (like YouTube's volume slider) which slides on horizontal wheel events without changing the scrolling behavior on vertical wheel events.  Since there is no equivalent of "touch-action" for wheel events, this case can only be implemented with non-passive wheel listeners."